### PR TITLE
[VPA] Add environment variable values to certGen container

### LINKED ISSUE #425: [vpa] Support HTTPS proxy in certgen Job
**Is your feature request related to a problem? Please describe.**

Currently when `admissionController.enabled=true` is enabled, by default certificates will generated as a `pre-install`/`pre-upgrade` hook in [`admission-controller-certgen.yaml`](https://github.com/FairwindsOps/charts/blob/master/stable/vpa/templates/admission-controller-certgen.yaml).

The `{{ include "vpa.fullname" . }}-certgen` Job requires public network access to [install packages](https://github.com/FairwindsOps/charts/blob/7464dcd2e8a430771557e05dacd95bbcd46f7fec/stable/vpa/templates/admission-controller-certgen.yaml#L76). In environments where public access is not allowed and a _HTTPS proxy_ must be used the Job fails:

```
fetch http://dl-cdn.alpinelinux.org/alpine/v3.10/main/x86_64/APKINDEX.tar.gz
ERROR: http://dl-cdn.alpinelinux.org/alpine/v3.10/main: network error (check Internet connection and firewall)
fetch http://dl-cdn.alpinelinux.org/alpine/v3.10/community/x86_64/APKINDEX.tar.gz
ERROR: http://dl-cdn.alpinelinux.org/alpine/v3.10/community: network error (check Internet connection and firewall)
(1/1) Installing openssl (1.1.1i-r0)
ERROR: openssl-1.1.1i-r0: network error (check Internet connection and firewall)
1 error; 81 MiB in 39 packages
```

**Describe the solution you'd like**

The ability to pass in environment variables and/or additional arguments to the `Job`'s container spec would allow `https_proxy=...` to be specified to overcome this.

### PR DESCRIPTION
**Pull request by sudermanjr**
**Why This PR?**

Fixes #425

**Checklist:**

* [x] I have updated the chart version in `Chart.yaml` following Semantic Versioning.
* [x] Any new values are backwards compatible and/or have sensible default.
* [x] Any new values have been added to the README for the Chart, or helm-docs has been run for the charts that support it.

### CODE REVIEW

File: stable/vpa/templates/admission-controller-certgen.yaml [hunk 1]
```diff
@@ -117,5 +117,10 @@ spec:
             # Clean up after we're done.
             echo "Deleting ${TMP_DIR}."
             rm -rf ${TMP_DIR}
+        env:
```
Comment (rbren) [inline stable/vpa/templates/admission-controller-certgen.yaml:120]: If there are no keys, will this still be OK?

**Review by rbren — COMMENTED** [review]
Looks like maybe an errant file called `stable/?`

File: stable/vpa/templates/admission-controller-certgen.yaml (hunk 1 above)
Comment (sudermanjr) [reply stable/vpa/templates/admission-controller-certgen.yaml:120]: Yeah, because the default values file specifies `{}`

Comment (rbren) [reply stable/vpa/templates/admission-controller-certgen.yaml:120]: Oh that's what I meant - if it's an empty obect, the yaml ends up looking like
```
foo: bar
env:
baz: quuz
```
Does yaml treat that as `null`?

**Review by rbren — APPROVED** [review]

File: stable/vpa/templates/admission-controller-certgen.yaml (hunk 1 above)
Comment (sudermanjr) [reply stable/vpa/templates/admission-controller-certgen.yaml:120]: Seems to be the case. Just tested it.

**Pull request merged**